### PR TITLE
[IOACGN-19] Introduces new CGN remote feature flag

### PR DESCRIPTION
**Pull request by CrisTofani**
This PR introduces CGN remote feature flag and configuration

### CODE REVIEW

**Review by Undermaken — CHANGES_REQUESTED** [review]
this PR LGTM
could you also update `packages.json` by increasing the version to `1.0.3` (the next tag)

when this PR will be merged could you also push the `1.0.3` on master?

File: definitions.yml [hunk 1]
```diff
@@ -427,6 +431,19 @@ definitions:
...
+    type: object
+    description: "dedicated config for the CGN app flow"
+    required:
+      - enabled
+      - cgn_merchants_v2
+    properties:
+      enabled:
+        type: boolean
+        description: "If true CGN app components and logic is available in those app that support it"
+      cgn_merchants_v2:
```
Comment (Undermaken) [inline definitions.yml:444]:
```suggestion
      merchants_v2:
```
since we already are in `cgn` section
it's just syntax sugar. What do you think?

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Comment by CrisTofani** [comment]
> this PR LGTM could you also update `packages.json` by increasing the version to `1.0.3` (the next tag)
> 
> when this PR will be merged could you also push the `1.0.3` on master?

@Undermaken fixes up, when will be merged on master I'll procede pushing the tag

File: definitions.yml [hunk 2]
```diff
@@ -427,6 +431,19 @@ definitions:
       enabled:
         type: boolean
         description: "If true paypal is available in those app that support it"
+  CgnConfig:
+    type: object
+    description: "dedicated config for the CGN app flow"
+    required:
+      - enabled
+      - cgn_merchants_v2
```
Comment (Undermaken) [inline definitions.yml:439]:
```suggestion
      - merchants_v2
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]

**Pull request merged**